src/components/Attachments/AttachmentView/DefaultAttachmentView/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import type {StyleProp, ViewStyle} from 'react-native';
+import {ActivityIndicator, View} from 'react-native';
+import Icon from '@components/Icon';
+import * as Expensicons from '@components/Icon/Expensicons';
+import Text from '@components/Text';
+import Tooltip from '@components/Tooltip';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+type DefaultAttachmentViewProps = {
+    /** The name of the file */
+    fileName?: string;
+    /** Should show the download icon */
+    shouldShowDownloadIcon?: boolean;
+    /** Should show the loading spinner icon */
+    shouldShowLoadingSpinnerIcon?: boolean;
+    /** Additional styles for the container */
+    containerStyles?: StyleProp<ViewStyle>;
```

Comment:
NAB: Add newlines between these 
```suggestion
    /** The name of the file */
    fileName?: string;
    
    /** Should show the download icon */
    shouldShowDownloadIcon?: boolean;

    /** Should show the loading spinner icon */
    shouldShowLoadingSpinnerIcon?: boolean;

    /** Additional styles for the container */
    containerStyles?: StyleProp<ViewStyle>;
```